proto2python.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Bash script to create the python packages from the grpc proto for Armonik.Api
+# We are using the same structure than the C# package
+
+export PATH=$HOME/.local/bin:$PATH
+
+export PROTO_PATH="Protos/V1"
+export ARMONIK_PYTHON_SRC="Api/python/src" 
+export ARMONIK_SERVER=$ARMONIK_PYTHON_SRC"/armonik/server"
```

Comment:
Naming is ambiguous
Maybe WORKER would be better. This is for the worker side of ArmoniK but has both grpc client and server components

Reply:
I am using the same naming thant the extension CSharp (in theory)

Reply:
In extension.csharp, it's worker